### PR TITLE
community/dnscrypt-proxy: upgrade to 2.0.23

### PR DESCRIPTION
**Pull request by ianbashford**
.onion servers are now automatically ignored if Tor routing is not enabled
caching of server addresses has been improved, especially when using proxies.
DNSCrypt communications are now automatically forced to using TCP when a SOCKS proxy has been set up.

### CODE REVIEW

File: community/dnscrypt-proxy/APKBUILD [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Contributor: Ian Bashford <ianbashford@gmail.com>
 # Maintainer: Ian Bashford <ianbashford@gmail.com>
 pkgname=dnscrypt-proxy
-pkgver=2.0.22
+pkgver=2.0.23
 pkgrel=0
 pkgdesc="A tool for securing communications between a client and a DNS resolver"
 url="https://dnscrypt.info"
```
Comment (maxice8) [inline community/dnscrypt-proxy/APKBUILD]: i think the openrc subpackage is automatically made into noarch by abuild so it is not required to turn it into it via ::noarch

Comment (tcely) [reply community/dnscrypt-proxy/APKBUILD]: That is correct

```
case "$subpkgname" in
	*-doc | *-openrc | *-lang | *-lang-*) subpkgarch="noarch" ;;
	*) subpkgarch="$pkgarch" ;;
esac
```

**Comment by ianbashford** [comment]
Thanks for the tweak -- do you need me to submit a changed PR or do anything else?

**Comment by tcely** [comment]
It would be great if you would rebase the branch and squash / fixup the tweak so there is only one commit.

**Comment by ianbashford** [comment]
All done @tcely

**Pull request closed without merging**